### ISSUE TITLE
SteamVR beta motion reprojection uodate crashes OVRAS

### ISSUE THREAD

**Issue by d1360** [matzman666/OpenVR-AdvancedSettings#157]
You can't click anything on the tab after this great steamvr beta update

**Comment by Nicksonman** [comment]
Agreed, OpenVR AdvancedSettings is no longer usable since this update.

**Comment by Synkndown** [comment]
Advanced setting is still fully usable, however up and down are inverted

**Comment by Synkndown** [comment]
Inverting playspace or floor fix are both odd, and a room scale setup had to be redone to fix.

**Comment by Nicksonman** [comment]
Sounds like everyone got affected a little differently.

When I open any tab on ovas I get stuck on that tab - the back arrow doesn't work.

Even if I get out of ovas and go back to it, it's still stuck on the last tab I was on. 

Given that I couldn't roll back the steamvr beta update, the solution was to simply opt out of the beta, now my ovas works as normal.

**Comment by Synkndown** [comment]
Looks like steam just fixed the upside down inverted cursor thing with the new update
